mico-admin/src/app/data-view/mico-data-container/mico-data-container.component.ts

```diff
@@ -51,6 +51,10 @@ export class MicoDataContainerComponent implements OnChanges {
     propertyType(): string {
         if (this.property != null && this.property.type != null) {
             return this.property.type;
-        } else { return 'string'; }
+        } else if (this.property != null && this.property.$ref != null) {
+            return 'object';
+        } else {
+            return 'string';
```

Comment:
maybe add a json case to mico data to display unknown data as json instead of assuming it is a string